lib/manager/npm/post-update/npm.ts

```diff
@@ -139,6 +139,15 @@ export async function generateLockFile(
     }
     const duration = process.hrtime(startTime);
     const seconds = Math.round(duration[0] + duration[1] / 1e9);
+    if (
+      filename === 'npm-shrinkwrap.json' &&
+      pathExistsSync(join(cwd, 'package-lock.json'))
```

Comment:
Much preferred if we can await an async path exists function here instead of a sync one

Reply:
https://github.com/renovatebot/renovate/pull/4776/commits/582955ad52a38bf733fd6c8e0cd141c7a400892d